### DISCUSSION TITLE
How to delete/kill other active sessions within session manager?

### DISCUSSION THREAD

**Discussion by Andrew15-5** [zellij-org/zellij#3016] [Q&A] [answered]
I opened 3 terminals and then fiddle around. Then I have 3 sessions, one of which is current. Why I only can delete exited sessions? How to delete non-current active sessions? Do I have to use `zellij kill-all-sessions`?

**Comment by Andrew15-5** [comment]
I only found out that I can kill the current session with `{ CloseFocus; SwitchToMode "Normal"; }` in pane mode. After all panes were closed, I'm dropped into another active session. But this is completely different and janky way. At least being able to delete the whole current session at once and drop into another active session would be a progress.

Update: I just tried doing that and it doesn't work. The terminal window just closes altogether instead of dropping into another active session.

**Comment by cristiand391** [comment, marked as answer]
Doesn't seem possible yet, `zellij 0.39.2` doesn't have a plugin command to kill a specific session:
https://github.com/zellij-org/zellij/blob/main/zellij-utils/src/data.rs#L1020-L1108

Not sure if there's a limitation about this (there's no CLI action available, just `zellij kill` commands, but that would be required to make it work in session-manager/zellij plugins.
